src/seeds.bioregion.cpp

```diff
@@ -231,6 +254,10 @@ void bioregion::remove_member(name account) {
     
     check(mitr != members.end(), "member not found");
 
+    roles_tables roles(get_self(), mitr -> bioregion.value);
+    auto ritr = roles.find(account.value);
+    check(ritr == roles.end(), "user can not leave bioregion because it has a role in it");
+
```

Comment:
Could either remove the role, or have them remove the role first - I am fine with this, good catch

Clients can automate this , e.g. they can check if there's roles, and add a leave the role action before the leave action

Reply:
Actually if a member is removed by an admin, they should also be removed from the role?! Hmm.

Reply:
Ok, it is better to have the role removed automatically